### PR TITLE
Fix boilerplate issues

### PR DESCRIPTION
**Pull request by pmorie**
Fixes boilerplate issues in various source files in preparation for repo-infra boilerplate enforcement.

### CODE REVIEW

**Comment by pmorie** [comment]
Gonna just merge this once CI passes.

File: hack/bookstore_client.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/python -u
```
Comment (pmorie) [inline hack/bookstore_client.py]: Good catch; i'm adding stuff to the boilerplate checker to be permissive on python shebangs.

Comment (pmorie) [inline hack/bookstore_client.py]: Fixed.

**Comment by MHBauer** [comment]
LGTM

**Comment by bmelville** [comment]
LGTM, feel free to merge when CI is green.

**Pull request merged**